README.rst

```diff
@@ -70,8 +70,8 @@ Getting Started
 Windows VM Software (versions given are ones I used, others likely work too):
 
 - Windows 7 Ultimate 32 bit
-- Dragon NaturallySpeaking Premium 12
-- NatLink 4.1echo
+- Dragon NaturallySpeaking Premium 12. (Version 13 has been experimented with recently and works well, with much better accuracy than 12, therefore we recommend using version 13. If you do have problems with the version 13 and find out how to solve them, please do update this documentation or add information to GitHub issue.)
```

Comment:
(optional) this is a bit cumbersome. Maybe now that we're describing multiple configurations, we should consider formatting this as a table, or as a list where each entry is a description of all versions that is working for one person. When I first wrote this, it was simply a list of what I was using and that worked. Perhaps we should add additional such lists for other people who have them working (and, strictly opt in, identify them for followup if they're open to it?)

I just got one thing working and then never updated it so I welcome feedback from others who have more experience here.

Anyway I don't think this needs to block this PR.

Reply:
I agree that's a bit of a cumbersome read. I will just shorten the text in the parentheses since I will be playing with Dragon 13 a lot, and will report/try to fix whatever bugs I find. I guess I haven't found any so far.